lib/elixir/lib/module.ex

```diff
@@ -1169,13 +1183,16 @@ defmodule Module do
   end
 
   @doc false
-  def check_behaviours_and_impls(env, table, all_definitions) do
+  def check_behaviours_and_impls(env, table, all_definitions, overridable_pairs_not_overridden) do
     behaviours = :ets.lookup_element(table, :behaviour, 2)
     impls = :ets.lookup_element(table, {:elixir, :impls}, 2)
+    definitions_to_check = for definition = {pair, _, _, _} <- all_definitions,
```

Comment:
Please pass `overridable_pairs_not_overridden` to inside `warn_missing_impls` and you can check for it inside the loop that already traverses all definition pairs.